/**:
  ros__parameters:
    # Plugin execution order - plugins will run in this order
    plugin_names:
      - "autoware::trajectory_optimizer::plugin::TrajectoryPointFixer"
      - "autoware::trajectory_optimizer::plugin::TrajectoryKinematicFeasibilityEnforcer"
      - "autoware::trajectory_optimizer::plugin::TrajectoryQPSmoother"
      - "autoware::trajectory_optimizer::plugin::TrajectoryKinematicFeasibilityEnforcer"
      - "autoware::trajectory_optimizer::plugin::TrajectoryEBSmootherOptimizer"
      - "autoware::trajectory_optimizer::plugin::TrajectorySplineSmoother"
      - "autoware::trajectory_optimizer::plugin::TrajectoryMPTOptimizer"
      - "autoware::trajectory_optimizer::plugin::TrajectoryVelocityOptimizer"
      - "autoware::trajectory_optimizer::plugin::TrajectoryExtender"

    # Plugin activation flags - control runtime enable/disable
    use_akima_spline_interpolation: true
    use_eb_smoother: false
    use_qp_smoother: true
    use_trajectory_point_fixer: true
    use_velocity_optimizer: true
    use_trajectory_extender: false
    use_kinematic_feasibility_enforcer: true
    use_mpt_optimizer: false
